### PR TITLE
Update plans and pricing page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Updated the `/support/plans-and-pricing` page with the latest changes and some table highlighting.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Check the content matches the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#)
- Check that the new table cell highlight is ok

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2275

### CODE REVIEW

**Comment by anthonydillon** [comment]
@frankiegranato thanks. I have actioned all your comments. It is ready for another review when you have time.

> I’ve moved the ‘Optional add ons’ to below “For Desktops”. See copydoc.

Done.

> The ‘For desktops’ table ends quite abruptly - is something wrong with the styling here?

I have updated the layout of the table. Believe its must better now.

> Re the ’OpenStack’ table - can we add some extra space around the Free column? It very tight looking. Should Karlos review this?

I have updated the layout of this table to give it all some more room. There is a design needed label on this PR so a design will need to review this before landing.

> Re The texts with asterixis * - Can the text be smaller please?

Done

> The table for Ubuntu Advantage for Kubernetes needs a design review. It looks a bit fiddly. If you don’t centre the “Ubuntu Advantage for Kubernetes” heading, and break after ‘ubuntu Advantage’, it might look better, but I think Karlos should review.

Updated the layout of the table and will see what the design review comes back with.

> RE: "* Ongoing enterprise support for Kubernetes after 30 days is optional." - I have removed the link here.

Done.

File: templates/support/plans-and-pricing.html [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <div class="col-4 p-card">
       <h3>Consulting packages</h3>
       <div class="p-card__footer">
-        <p class="p-heading--five">From $150 per year <span style="font-size: 1rem;">*</span></p>
+        <p class="p-heading--five">From $2500 per day <span style="font-size: 1rem;">*</span></p>
```
Comment (pmahnke) [inline templates/support/plans-and-pricing.html]: should have a comma.... $2,500

File: templates/support/plans-and-pricing.html [hunk 2]
```diff
@@ -148,14 +147,14 @@
...
-      <p>Canonical provides support services, management tools, and the Cloud Archive which keeps thousands of businesses productive with their own local OpenStack clouds.  For large volume and public clouds with dynamic scale, we’re pleased to offer per-region pricing with flexible node ranges.</p>
+      <h2>OpenStack</h2>
+      <p>Canonical provides support services, management tools, and the Cloud Archive which keeps thousands of businesses productive with their own local OpenStack clouds.  For large volume and public clouds with dynamic scale, we&rsquo;re pleased to offer per-region pricing with flexible node ranges.</p>
     </div>
   </div>
   {% include "shared/pricing/_openstack-support.html" %}
   <div class="row">
     <div class="col-8">
-      <p><a href="/support/contact-us?product=cloud-plans-and-pricing">Buy Ubuntu Advantage&nbsp;&rsaquo;</a></p>
+      <p><a href="https://buy.ubuntu.com/">Buy Ubuntu Advantage&nbsp;&rsaquo;</a></p>
```
Comment (pmahnke) [inline templates/support/plans-and-pricing.html]: external link?

**Comment by frankiegranato** [comment]
Looks great. Nice job. One tiny thing that I’m sure Karlos will pick up on anyway - at the bottom of the OpenStack section, I would close up the space between the two asterixised sentences, but this is not a showstopper by any means. I’m going to add my +1 now.

**Comment by anthonydillon** [comment]
@pmahnke @frankiegranato All done, ready for another review.

**Pull request merged**